backend/query/src/items/interfaces/query-item-list.dto.ts

```diff
@@ -36,14 +36,16 @@ export class QueryItemListDto {
 
   @IsString()
   @IsOptional()
-  @Length(2, 100)
+  @Length(0, 100)
   @ApiPropertyOptional()
   query: string;
 
   @IsString({ each: true })
   @IsArray()
   @IsOptional()
   @ApiPropertyOptional()
+  @Type(() => String)
+  @Transform(({ value }) => (Array.isArray(value) ? value : [value]))
```

Comment:
I think you can replace this with `[value].flat()`. Should to the same and is way shorter